mmv1/templates/terraform/post_create/labels.erb

```diff
@@ -1,13 +1,14 @@
 <% if properties.any?{ |p| p.name == "labels" } -%>
 if v, ok := d.GetOkExists("labels"); !tpgresource.IsEmptyValue(reflect.ValueOf(v)) && (ok || !reflect.DeepEqual(v, labelsProp)) {
 	// Labels cannot be set in a create.  We'll have to set them here.
-	err = resource<%= resource_name -%>Read(d, meta)
+	// Only labelFingerprint is needed in the request to set labels.
+	err = resource<%= resource_name -%>LabelFingerprintRead(d, meta)
```

Comment:
Instead of needing a new function could we `d.Get` `labels`, hold the value, call Read, and then `d.Set` `labels` to update the state?

Reply:
Yeah, this is a better and safer option.